src/configure

```diff
@@ -1410,6 +1412,7 @@ Optional Features:
   --enable-experimental-code
                           Use experimental code
   --enable-fuzz           Fuzzing prepare(), valid(), init(), etc
+  --enable-fuzzing        Fuzzing using libFuzzer
```

Comment:
Can I call it `--enable-libfuzzer` instead of `--enable-libfuzz`? The former name sounds closer to the software (i.e. libFuzzer) we are using for fuzzing.